src/parser/jobs/mnk/Fists.js

```diff
@@ -60,4 +78,11 @@ export default class Fists extends Module {
 			value: this._stanceless,
 		}))
 	}
+
+	getBuffUptimePercent(statusId) {
+		const statusUptime = this.combatants.getStatusUptime(statusId, this.parser.player.id)
+		const fightUptime = this.parser.fightDuration - this.invuln.getInvulnerableUptime()
```

Comment:
Unless `this.combatants.getStatusUptime()` filters out invuln windows, I don't think you need (or actually want) to exclude them from the overall fight duration, since MNK Fist buffs don't rely on hitting things (unlike just about every other buff of their ilk that I can think of). They're just... active.

Reply:
I actually didn't think of that and just copypasted from the other main module in this PR. Downtime is often enough FoE anyway though hmm. I'll adjust this one.